packages/zql/src/query/query-impl.ts

```diff
@@ -745,8 +745,7 @@ export class QueryImpl<
     let queryComplete = delegate.defaultQueryComplete;
     const gotCallback: GotCallback = got => {
       if (got) {
-        const t1 = Date.now();
-        delegate.onQueryMaterialized(this.hash(), ast, t1 - t0);
+        delegate.onQueryMaterialized(this.hash(), ast, performance.now() - t0);
```

Comment:
this is a very misleading metric in that it will include the time it took the server to hydrate all queries sent in a `changeDesiredQueries` message.

wdyt about removing it while you're here and exposing a different metric like a "changeDesiredQueries histogram"?